packages/flame/lib/src/game/game.dart

```diff
@@ -159,6 +160,18 @@ abstract class Game {
     return _gameRenderBox!.localToGlobal(point.toOffset()).toVector2();
   }
 
+  @override
+  Vector2 unprojectVector(Vector2 vector) => vector;
+
+  @override
+  Vector2 projectVector(Vector2 vector) => vector;
+
+  @override
+  Vector2 unprojectDelta(Vector2 vector) => vector;
+
+  @override
+  Vector2 projectDelta(Vector2 vector) => vector;
```

Comment:
These methods need some dart doc

Reply:
Dont they inherit the dartdoc from Projector?

Reply:
I added the docs centralized on the interface

Reply:
@wolfenrain yes, I tested that, the dartdocs show fine on the IDE

Reply:
Sounds good, didn't realized this came from an interface.